server/sonar-server/src/test/java/org/sonar/server/rule/index/RuleIndexDefinitionTest.java

```diff
@@ -100,13 +104,13 @@ public void sanitize_html_description_as_it_is_english() {
     // "this", "is", "a" and "of" are not indexed.
     // Plural "words" is converted to singular "word"
     List<AnalyzeResponse.AnalyzeToken> tokens = analyzeIndexedTokens(text);
-    assertThat(tokens).extracting("term").containsOnly("small", "list", "word");
+    assertThat(tokens).extracting("term").containsOnly("small", "list", "words");
```

Comment:
the comment `Plural "words" is converted to singular "word"` is no more relevant ?

Reply:
Removed.